change/@internal-react-components-6507a903-0818-4048-8755-ef850d53c2bf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add tab index to local video tile through a wrapper to allow keyboard navigation of the modal controls.",
```

Comment:
After merging in from main this should just be updating the string as main already has the tabindex property set